tests/database/migrations/2024_07_15_000009_create_testbench_post_statuses_table.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+final class CreateTestbenchPostStatusesTable extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('post_statuses', function (Blueprint $table): void {
+            $table->increments('id');
+            $table->enum('status' , ["DONE" , "PENDING"]);
+            $table->unsignedInteger("post_id")
```

Comment:
why mix single quotes and double quotes?

Reply:
fixed